src/cwt/mod.rs

```diff
@@ -193,7 +206,7 @@ mod test {
             .expect("claims_set2: failed to insert issuer");
 
         let serialized3 =
-            hex::decode("a23a000f423ffbc059161e4f765fd967746573746b6579393038").unwrap();
+            hex::decode("a267746573746b65793930383a000f423ffbc059161e4f765fd9").unwrap();
```

Comment:
These are the same CBOR structures with map elements in a different order. In the future we could update this test to not rely on a specific ordering.